### PR TITLE
Fixes and general updates

### PR DESCRIPTION
**Pull request by laijasmine**
addresses issues: #128, #126, #125, #122, #121, #120, #119 

- Mainly adding more note blocks
- Making necessary updates to material to reflect new changes in software
- Moves Nesting to reference guide
- Clarifies points of confusion

### CODE REVIEW

File: training/04_editing_eml.Rmd [hunk 1]
```diff
@@ -20,6 +24,11 @@ This chapter is a practical tutorial for using R to read, edit, write, and valid
 ```{r, child = '../workflows/edit_eml/edit_otherentities.Rmd'}
 ```
 
+## Exercise 3a {.exercise}
+
+* Make sure your package from [before](#exercise-2) is loaded into R.
+* Replace the existing `dataTable` with a new `dataTable` object with an `attributelist` and `physical` section you write in R using the above commands.
```
Comment (jeanetteclark) [inline training/04_editing_eml.Rmd:30]: are we suggesting that people update their package on the node before proceeding to 3b? If so we should be explicit about that

File: workflows/edit_eml/edit_attributelists.Rmd [hunk 2]
```diff
@@ -83,8 +83,9 @@ EML::shiny_attributes(attributes_table = attributes_table)
 EML::shiny_attributes()
```
Comment (jeanetteclark) [inline workflows/edit_eml/edit_attributelists.Rmd:83]:
```suggestion
atts <- EML::shiny_attributes()
```

File: workflows/edit_eml/edit_attributelists.Rmd [hunk 3]
```diff
@@ -83,8 +83,9 @@ EML::shiny_attributes(attributes_table = attributes_table)
...
 ```
 
-Once you are done editing a table in the app, click the `Print` button to print text of a code that will build a data.frame in R. Copy that code and assign it to a variable in your script (e.g. `attributes <- data.frame(...)`).
+Once you are done editing a table in the app, quit the app and the tables will be returned as a list. Alternatively, each table can be to exported to a csv file by clicking the `Download` button.
```
Comment (jeanetteclark) [inline workflows/edit_eml/edit_attributelists.Rmd:86]:
```suggestion
Once you are done editing a table in the app, quit the app and the tables will be assigned to the `atts` variable as a list of data frames (one for attributes, factors, and units). Alternatively, each table can be to exported to a csv file by clicking the `Download` button.
```

File: workflows/edit_eml/edit_attributelists.Rmd [hunk 4]
```diff
@@ -83,8 +83,9 @@ EML::shiny_attributes(attributes_table = attributes_table)
...
 
+Read the table back into your R session and assign it to a variable in your script (e.g. `attributes <- data.frame(...)`).
```
Comment (jeanetteclark) [inline workflows/edit_eml/edit_attributelists.Rmd:88]:
```suggestion
If you downloaded the table, read the table back into your R session and assign it to a variable in your script (e.g. `attributes <- data.frame(...)`), or just use the variable that `shiny_attributes` returned.
```

File: workflows/pi_correspondence/navigate_rt.Rmd [hunk 5]
```diff
@@ -47,10 +47,25 @@ This is the queue interface from number 6 of the Front page
...
  + Comment - attach internal message (no `submitter`s, only Data Teamers)  
  + Open It - Open the ticket  
  + Stall - `submitter` has not responded in greater than 1 month  
  + Resolve - ticket completed  
 8. History - message history and option to reply (to `submitter` and beyond) or comment (internal message)
+
+### New data submission
+When notified by Arcticbot about a new data submission, here are the typical steps:
+
+1. Update the Requestor under the People section based on the email given in the submission (usually the user/ PI/ `submitter`)
```
Comment (jeanetteclark) [inline workflows/pi_correspondence/navigate_rt.Rmd:67]:
```suggestion
1. Update the Requestor under the People section based on the email given in the submission (usually the user/ PI/ `submitter`). You may have to google for the e-mail address if the PI did not include it in the metadata record.
```

File: workflows/pi_correspondence/navigate_rt.Rmd [hunk 6]
```diff
@@ -47,10 +47,25 @@ This is the queue interface from number 6 of the Front page
...
+2. Take the ticket
```
Comment (jeanetteclark) [inline workflows/pi_correspondence/navigate_rt.Rmd:68]:
```suggestion
2. Take the ticket (Actions > Take)
```

File: training/04_editing_eml.Rmd (hunk 1 above)
Comment (laijasmine) [reply training/04_editing_eml.Rmd:30]: hmmm I was thinking that 3a would be focusing on updating datatable and physical in this section and 3b is a continuation where you would be using what you made earlier to update the package... I've pushed a change that hopefully makes the language a bit more clear

also did not know that you can suggest changes... so cool!

Comment (jeanetteclark) [reply training/04_editing_eml.Rmd:30]: I know, its one of the reasons I love the PR model of making changes

**Pull request merged**